src/Dom/Document.php

```diff
@@ -355,6 +329,21 @@ public function __construct($version = '', $encoding = null)
         parent::__construct($version ?: '1.0', Encoding::AMP);
         $this->registerNodeClass(DOMElement::class, Element::class);
         $this->options = Option::DEFAULTS;
+
+        $this->registerMiddlewareClasses([
+            AmpEmojiAttribute::class,
+            AmpBindAttributes::class,
+            SelfClosingTags::class,
+            NoscriptElements::class,
+            DoctypeNode::class,
+            MustacheScriptTemplates::class,
+            CharacterEncoding::class,
+            HttpEquivCharset::class,
+            LoadHtmlWithLibXmlFlags::class,
+            NormalizeHtmlAttributes::class,
+            RemoveHttpEquivCharset::class,
+            SvgSourceAttributeEncoding::class,
```

Comment:
To avoid manual imports for every single one of these, we can use relative namespaces:
```suggestion
            Middleware\AmpEmojiAttribute::class,
            Middleware\AmpBindAttributes::class,
            Middleware\SelfClosingTags::class,
            Middleware\NoscriptElements::class,
            Middleware\DoctypeNode::class,
            Middleware\MustacheScriptTemplates::class,
            Middleware\CharacterEncoding::class,
            Middleware\HttpEquivCharset::class,
            Middleware\LoadHtmlWithLibXmlFlags::class,
            Middleware\NormalizeHtmlAttributes::class,
            Middleware\RemoveHttpEquivCharset::class,
            Middleware\SvgSourceAttributeEncoding::class,
```